charts/datadog/values.yaml

```diff
@@ -697,6 +697,9 @@ clusterAgent:
     # clusterAgent.image.tag -- Cluster Agent image tag to use
     tag: 1.22.0
 
+    # clusterAgent.image.digest -- Cluster Agent image digest to use
```

Comment:
It can be good to say that it takes precedence on the `tag`.
can be added also to other images `digest`

Reply:
Added requested changes